### DISCUSSION TITLE
Welcome to OpenRCT3 Discussions!

### DISCUSSION THREAD

**Discussion by chances** [open-rct3/open-rct3#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with community members. We hope that you:
  - Ask [questions](https://github.com/open-rct3/open-rct3/discussions/categories/q-a)
  - Share your [ideas](https://github.com/open-rct3/open-rct3/discussions/categories/ideas)
  - Engage with our community, and above all
  - Welcome others and are open-minded

Remember, this is a community we build together.

**Comment by Lzcky** [comment]
Did you give up with openrct3?

**Reply by chances** [reply]
We have not given up on the project.

<img width="626" height="413" alt="image" src="https://github.com/user-attachments/assets/6e7f365f-a153-4249-af99-05c13183644c" />

Rendering the `nullbmp`.

---

I'm sorry progress has been so slow, but this is one of the most complex software engineering challenges of my career. 

I've been making more of an effort lately to engage with the wider RCT3 community. Going forward, I'll do my best to keep everyone updated on the latest developments.

**Comment by TheRedhead43** [comment]
Rooting for you (and any other devs if any) to get this project done! If OpenRCT2 can exist and thrive, so can RCT3! Excited for the revival of the community (but patient)! :)
